sys/shell/commands/sc_vfs.c

```diff
@@ -200,9 +201,16 @@ static int _read_handler(int argc, char **argv)
             }
             printf("%02x", buf[k]);
         }
+        for (int k = res; k < sizeof(buf); ++k) {
+            if ((k % 2) == 0) {
+                putchar(' ');
+            }
+            putchar(' ');
+            putchar(' ');
+        }
         putchar(' ');
         putchar(' ');
-        for (int k = 0; k < res; ++k) {
+        for (int k = 0; k < sizeof(buf); ++k) {
```

Comment:
are the trailing dots really desired?
In my opinion the corrected padding was all that is needed. `xxd` on Linux for example does not add any trailing dots.